### PR TITLE
PullRequest for Issue2025: change the energy range for SXRMB

### PR DESCRIPTION
**Pull request by XuetaoLiu-CLS**
change the energy range from 1700~10000 to 1300 ~ 10000

#2025

### CODE REVIEW

**Comment by XuetaoLiu-CLS** [comment]
@helfrij can you check this one? Thx

File: source/application/SXRMB/SXRMBAppController.cpp [hunk 1]
```diff
@@ -348,6 +348,8 @@ void SXRMBAppController::setupExporterOptions()
 
 void SXRMBAppController::setupUserInterface()
 {
+	SXRMBBeamline *sxrmbBl = SXRMBBeamline::sxrmb();
```
Comment (helfrij) [inline source/application/SXRMB/SXRMBAppController.cpp]: Initializing the beamline object happens in startup() for BioXAS. Just checking--is this different for SXRMB?

Comment (helfrij) [inline source/application/SXRMB/SXRMBAppController.cpp]: Oh, 'kay. I see. You're using the sxrmbBl object below. :)

**Comment by helfrij** [comment]
Looks good!

Time Estimate: 5 minutes.

File: source/beamline/SXRMB/SXRMBBeamline.h [hunk 2]
```diff
@@ -64,6 +64,11 @@ class SXRMBBeamline : public CLSBeamline
 	/// Destructor.
 	virtual ~SXRMBBeamline();
 
+	/// returns the low end energy level
+	double beamlineEnergyLowEnd() { return beamlineEnergyLowEnd_; }
```
Comment (dretrex) [inline source/beamline/SXRMB/SXRMBBeamline.h]: These should be `const` functions.

File: source/beamline/SXRMB/SXRMBBeamline.cpp [hunk 3]
```diff
@@ -33,6 +33,8 @@ along with Acquaman.  If not, see <http://www.gnu.org/licenses/>.
 SXRMBBeamline::SXRMBBeamline()
 	: CLSBeamline("SXRMB Beamline")
 {
+	beamlineEnergyLowEnd_ = 1300;   //   --- Al 1.486
```
Comment (dretrex) [inline source/beamline/SXRMB/SXRMBBeamline.cpp]: The comment doesn't make sense.  1.486 keV? 1486 eV?

**Comment by dretrex** [comment]
Time Estimate: 2 minutes

**Comment by dretrex** [comment]
Looks like this has been good for a long time.  Sorry about that @XuetaoLiu-CLS.

**Pull request merged**